### PR TITLE
docs: added missing sub_filters in nginx subpath

### PR DESCRIPTION
**Pull request by C4J3**
Fix posters not showing up when using sub-folder (at least with NPM)

#### Description

Fixes title card images not appearing if using subfolders.
#### Screenshot (if UI-related)

Before:
image

After:
image

#### Aditional Notes

I know this is not officially supported but I figured this was a byproduct of the recent update and it just wasn't thought about. Small change but for idiots like me it'll save 10-100~ mins of trying to troubleshoot 😅.

### CODE REVIEW

File: docs/extending-jellyseerr/reverse-proxy.mdx [hunk 1]
```diff
@@ -95,6 +95,7 @@ location ^~ /jellyseerr {
     sub_filter '/api/v1' '/$app/api/v1';
     sub_filter '/login/plex/loading' '/$app/login/plex/loading';
     sub_filter '/images/' '/$app/images/';
+    sub_filter '/imageproxy/' '/$app/imageproxy/';
```
Comment (gauthier-th) [inline docs/extending-jellyseerr/reverse-proxy.mdx:98]: You should also add `avatarproxy` which was added in v2.
```suggestion
    sub_filter '/imageproxy/' '/$app/imageproxy/';
    sub_filter '/avatarproxy/' '/$app/avatarproxy/';
```

**Review by gauthier-th — CHANGES_REQUESTED** [review]

**Comment by C4J3** [comment]
Possibly unrelated to this. Is the subpath not meant to be maintained for actual jellyseerr subpaths? So if I have mydomain.tld/jellyseer I go to the discovery page but if I click movies I go to mydomain.tld/discover/movies. Don't remeber if I had this behaviour previously because I mostly used requestrr and didn't use external address when connecting to Jellyseerr itself.

**Comment by Fallenbagel** [comment]
> Possibly unrelated to this. Is the subpath not meant to be maintained for actual jellyseerr subpaths? So if I have mydomain.tld/jellyseer I go to the discovery page but if I click movies I go to mydomain.tld/discover/movies. Don't remeber if I had this behaviour previously because I mostly used requestrr and didn't use external address when connecting to Jellyseerr itself.

Subpath is not supported officially.

**Comment by Fallenbagel** [comment]
Also the commit message has to start from `docs: .....` or this won't be reflected on the docs after merge

**Review by Fallenbagel — APPROVED** [review]

**Comment by gauthier-th** [comment]
@all-contributors please add @C4J3 for doc

**Comment by Fallenbagel** [comment]
@all-contributors please add @C4J3 for doc

**Pull request merged**